src/brokencompass.js

```diff
@@ -1,5 +1,6 @@
 import { BCActor } from "./module/actor/BCActor";
 import { BCActorSheet } from "./module/actor/sheets/BCActorSheet";
+import "./module/chat/chat-hooks";
```

Comment:
Hier importierst du die ganze Datei, was zum Beispiel dafür sorgt, dass die `chatListeners()` Funktion im globalen Namensraum landet. Ich weiß nicht welches der bessere Weg ist, aber alternativ könnte man an dieser Stelle die Funktionen/Klassen in "/module/chat/chat-hooks.js" exportieren und hier dann entsprechend mit ihrem Namen/Namespace importieren. 

Ich mache sowas hier beispielsweise als Klasse: https://github.com/anvil-vtt/FateX/blob/master/src/module/features/ActorGroupFeature.ts#L9

Bin da aber auch nicht zufrieden, dass ich statische Methoden nutze. Wir könnten ja nochmal drüber nachdenken, welche Vor- und Nachteile wir jeweils hätten

Reply:
Es ist ist alles noch mit der heißen Nadel gestrickt. Wenn wir mit dem Setup fertig sind, muss das alles noch refaktort werden. Hier wäre es schön wenn wir uns Coding-Guidelines aufschreiben